zio-kafka-bench/src/main/scala/zio/kafka/bench/ConsumersComparisonBenchmark.scala

```diff
@@ -91,6 +92,25 @@ class ConsumersComparisonBenchmark extends ZioBenchmark[Env] {
       }
     }
 
+  @Benchmark
+  @BenchmarkMode(Array(Mode.AverageTime))
+  def kafkaClientsWithPrefetch(): Any =
+    runZIO {
+      for {
+        settings <- ZIO.service[ConsumerSettings]
+        consumer <- ZIO.service[LowLevelKafka]
+        _        <- ZIO.attemptBlocking(consumer.subscribe(java.util.Arrays.asList(topic1)))
+        _ <- ZStream
+               .repeatZIO(ZIO.attemptBlocking(consumer.poll(settings.pollTimeout)))
+               .map(records => Chunk.fromIterable(records.asScala))
+               .bufferChunks(if (settings.perPartitionChunkPrefetch > 1) settings.perPartitionChunkPrefetch else 16)
+               .flattenChunks
+               .take(nrMessages.toLong)
```

Comment:
This will produce better performances:

```suggestion
        _ <- ZStream
               .repeatZIOChunk(ZIO.attemptBlocking(Chunk.fromIterable(consumer.poll(settings.pollTimeout).asScala)))
               .bufferChunks(if (settings.perPartitionChunkPrefetch > 1) settings.perPartitionChunkPrefetch else 16)
               .take(nrMessages.toLong)
```